scripts/generate-grammar.vsh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env -S v
```

Comment:
You'll also need to update the Makefile that points to this. Make sure `make grammar` is working

Reply:
Done